### PR TITLE
Translation - French: Improvement

### PR DESCRIPTION
**Pull request by MisterClems**
Improvement of the french translation and translation of the new line (Round end screen)

### CODE REVIEW

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 1]
```diff
@@ -49,7 +49,7 @@ L.credit_kill = "Vous avez reçu {num} crédit(s) pour avoir tué un {role}."
 
 --- Karma
 L.karma_dmg_full = "Votre Karma est à {amount}, vous infligerez donc des dégâts normaux ce round!"
-L.karma_dmg_other = "Votre Karma est à {amount}. De ce fait tous les dégâts que vous infligerez seront réduits de {num}%"
+L.karma_dmg_other = "Votre Karma est à {amount}. De ce fait , tous les dégâts que vous infligerez seront réduits de {num}%"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:52]:
```suggestion
L.karma_dmg_other = "Votre Karma est à {amount}. De ce fait, tous les dégâts que vous infligerez seront réduits de {num}%"
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 2]
```diff
@@ -301,18 +301,18 @@ L.vis_hint = "Appuyez sur {usekey} pour ramasser (Détectives seulement)."
...
 -- Decoy
 L.decoy_name = "Leurre"
 L.decoy_no_room = "Vous ne pouvez pas prendre ce leurre."
-L.decoy_broken = "Votre Leurre a été détruit !"
+L.decoy_broken = "Votre leurre a été détruit !"
 
-L.decoy_short_desc = "Ce leurre montre un faux signal radar visible par les autres team"
-L.decoy_pickup_wrong_team = "Vous ne pouvez pas le ramasser car il appartient à une autre team"
+L.decoy_short_desc = "Ce leurre montre un faux signal radar visible par les autres teams"
+L.decoy_pickup_wrong_team = "Vous ne pouvez pas le ramasser , car il appartient à une autre team"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:312]:
```suggestion
L.decoy_pickup_wrong_team = "Vous ne pouvez pas le ramasser, car il appartient à une autre team"
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 3]
```diff
@@ -1003,7 +1003,7 @@ L.tbut_default = "Par défaut"
 L.name_door = "Porte"
 L.door_open = "Appuyez sur [{usekey}] pour ouvrir la porte."
 L.door_close = "Appuyez sur [{usekey}] pour fermer la porte."
-L.door_locked = "Cette porte est verrouillé"
+L.door_locked = "Cette porte est verrouillée"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:1006]:
```suggestion
L.door_locked = "Cette porte est verrouillée."
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 4]
```diff
@@ -1236,100 +1236,100 @@ L.lang_server_default = "Langue du serveur par défaut"
...
+L.title_event_finish = "Le round est terminé"
+L.title_event_game = "Un nouveau round a commencé"
+L.title_event_kill = "Un joueur a été tué"
+L.title_event_respawn = "Un joueur est réapparu"
+L.title_event_rolechange = "Un joueur a changé de rôle ou de team"
+L.title_event_selected = "Les rôles ont été distribuer"
+L.title_event_spawn = "Un joueur est apparu"
+
+L.desc_event_bodyfound = "{finder} ({firole} / {fiteam}) a trouvé le corps de {found} ({forole} / {foteam}). Le cadavre avait {credits} crédit(s) d'équipement."
+L.desc_event_bodyfound_headshot = "Le joueur a été tué par un tir dans la tête.."
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:1292]:
```suggestion
L.desc_event_bodyfound_headshot = "Le joueur a été tué par un tir dans la tête."
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 5]
```diff
@@ -198,20 +198,20 @@ L.search_dmg_fall = "Une chute lui a été fatale."
...
 L.search_weapon = "Il semblerait qu'un {weapon} a été utilisé pour le tuer."
-L.search_head = "Le blessure fatale a été portée à la tête. Impossible de crier."
+L.search_head = "Une blessure fatale a été portée à la tête. Impossible de crier."
 L.search_time = "Il est mort {time} avant que vous fassiez l'enquête."
 L.search_dna = "Ramassez un échantillon de l'ADN du tueur avec un Scanner ADN. Cet échantillon va se décomposer dans à peu près {time}."
 
 L.search_kills1 = "Vous avez trouvé une liste de meurtres qui confirme la mort de {player}."
 L.search_kills2 = "Vous avez trouvé une liste de meurtres avec les noms suivants :"
-L.search_eyes = "En utilisant vos compétences de détectives, vous avez identifié la dernière personne qu'il a vu : {player}. Serait-ce le tueur, ou une coïncidence ?"
+L.search_eyes = "En utilisant vos compétences de détective, vous avez identifié la dernière personne qu'il a vue : {player}. Serait-ce le tueur, ou une coïncidence ?"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:214]:
```suggestion
L.search_eyes = "En utilisant vos compétences de détective, vous avez identifié la dernière personne qu'il a vue: {player}. Serait-ce le tueur, ou une coïncidence?"
```

Comment (Alf21) [reply gamemodes/terrortown/gamemode/shared/lang/fr.lua:214]: Same above with `:`

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 6]
```diff
@@ -186,7 +186,7 @@ L.search_role_traitor = "C'était un Traitre!"
 L.search_role_det = "C'était un Détective."
 L.search_role_inno = "C'était un Terroriste Innocent."
 
-L.search_words = "Quelque chose vous dit que quelques unes des dernières paroles de cette personne étaient : '{lastwords}'"
+L.search_words = "Quelque chose vous dit que quelques-unes des dernières paroles de cette personne étaient : '{lastwords}'"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:189]:
```suggestion
L.search_words = "Quelque chose vous dit que quelques-unes des dernières paroles de cette personne étaient: '{lastwords}'"
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 7]
```diff
@@ -62,7 +62,7 @@ L.body_found_inno = "C'était un Innocent."
 
 L.body_confirm = "{finder} a confirmé la mort de {victim}."
 
-L.body_call = "{player} Appelle un Détective sur le corps de {victim} !"
+L.body_call = "{player} appelle un Détective sur le corps de {victim} !"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:65]:
```suggestion
L.body_call = "{player} appelle un Détective sur le corps de {victim}!"
```

Same 3 times below

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 8]
```diff
@@ -31,12 +31,12 @@ L.round_started = "Le round a commencé !"
 L.round_restart = "Un admin a redémarré le round."
 
 L.round_traitors_one = "Traitre, vous êtes seul."
-L.round_traitors_more = "Traitre, voici vos alliés : {names}"
+L.round_traitors_more = "Traitres, voici vos alliés : {names}"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:34]:
```suggestion
L.round_traitors_more = "Traitres, voici vos alliés: {names}"
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 9]
```diff
@@ -31,12 +31,12 @@ L.round_started = "Le round a commencé !"
...
 
-L.win_time = "Temps écoulé. Les Traitres ont perdus."
-L.win_traitor = "Les Traitres ont gagnés !"
+L.win_time = "Temps écoulé. Les Traitres ont perdu."
+L.win_traitor = "Les Traitres ont gagné !"
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:37]:
```suggestion
L.win_traitor = "Les Traitres ont gagné!"
```

Same 2 times below

**Review by Alf21 — CHANGES_REQUESTED** [review]
Just some format related issues

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 10]
```diff
@@ -198,20 +198,20 @@ L.search_dmg_fall = "Une chute lui a été fatale."
...
 L.search_dmg_other = "Vous n'arrivez pas à identifier la cause de sa mort."
 
 L.search_weapon = "Il semblerait qu'un {weapon} a été utilisé pour le tuer."
-L.search_head = "Le blessure fatale a été portée à la tête. Impossible de crier."
+L.search_head = "Une blessure fatale a été portée à la tête. Impossible de crier."
 L.search_time = "Il est mort {time} avant que vous fassiez l'enquête."
 L.search_dna = "Ramassez un échantillon de l'ADN du tueur avec un Scanner ADN. Cet échantillon va se décomposer dans à peu près {time}."
 
 L.search_kills1 = "Vous avez trouvé une liste de meurtres qui confirme la mort de {player}."
 L.search_kills2 = "Vous avez trouvé une liste de meurtres avec les noms suivants :"
```
Comment (TimGoll) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:213]:
```suggestion
L.search_kills2 = "Vous avez trouvé une liste de meurtres avec les noms suivants:"
```

File: gamemodes/terrortown/gamemode/shared/lang/fr.lua [hunk 11]
```diff
@@ -517,15 +517,15 @@ Surveillez vos arrières, et bossez avec vos camarades pour vous sortir d'ici en
...
+Utilisez-les pour aider les innocent à survivre, mais attention :
 les traitres chercheront à vous tuer en premier !
 
 Appuyez sur {menukey} pour recevoir votre équipement !]]
 
 L.info_popup_traitor_alone = [[
 Vous êtes un TRAITRE ! Vous n'avez pas d'amis traitres ce round.
 
-Tuez tous le monde pour gagner !
+Tuez tout le monde pour gagner !
```
Comment (TimGoll) [inline gamemodes/terrortown/gamemode/shared/lang/fr.lua:528]:
```suggestion
Tuez tout le monde pour gagner!
```

**Review by TimGoll — COMMENTED** [review]
Looking through the lang, it seems like there is always a space in front of puctuation marks. Is this a thing in french? Or is this bad practice? While it isn't something that you introduced, we'd really appreciate it, if you could remove these spaces (at least if it is bad practice in French).
Just just Strg+F to find and replace

**Comment by MisterClems** [comment]
> Looking through the lang, it seems like there is always a space in front of puctuation marks. Is this a thing in french? Or is this bad practice? While it isn't something that you introduced, we'd really appreciate it, if you could remove these spaces (at least if it is bad practice in French).
> Just just Strg+F to find and replace

Yeah thats my bad gonna fix this

**Comment by MisterClems** [comment]
Okey is fix all should be good

**Review by TimGoll — APPROVED** [review]
Thanks, looks good to me

**Comment by TimGoll** [comment]
you probably want to add this to the changelog as well. Something like "updated french translation (by MisterClems)"

**Comment by MisterClems** [comment]
> you probably want to add this to the changelog as well. Something like "updated french translation (by MisterClems)"

Done.But not sure if i have write in on the good section .

**Pull request merged**